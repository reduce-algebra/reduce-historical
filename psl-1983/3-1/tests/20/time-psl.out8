PSL Spectral Tests,  DEC-20 test system,  No-Date-Yet, test8
---------------------------------------------------------------
*** Garbage collection starting NIL  3191 193 69
*** GC %w: time %d ms 2 489 18 6
*** %d recovered, %d stable, %d active, %d free 204 157 9413 1
EmptyTest 10000		18
SlowEmptyTest 10000	187
Cdr1Test 100		527
Cdr2Test 100		372
CddrTest 100		274
ListOnlyCdrTest1	1769
ListOnlyCddrTest1	3194
ListOnlyCdrTest2	2790
ListOnlyCddrTest2	4083
ReverseTest 10		458
*** Garbage collection starting NIL  3191 193 77
*** GC %w: time %d ms 3 1071 5669 5656
*** %d recovered, %d stable, %d active, %d free 37096 9533 37 1
MyReverse1Test 10	458
*** Garbage collection starting NIL  3191 193 77
*** GC %w: time %d ms 4 1064 5237 5224
*** %d recovered, %d stable, %d active, %d free 36664 9533 37 1
MyReverse2Test 10	441
*** Garbage collection starting NIL  3191 193 76
*** GC %w: time %d ms 5 1063 5237 5224
*** %d recovered, %d stable, %d active, %d free 36664 9533 37 1
LengthTest 100		560
ArithmeticTest 10000	643
EvalTest 10000		2434
tak 18 12 6		476
gtak 18 12 6		1378
gtsta g0		1132
gtsta g1		1195
