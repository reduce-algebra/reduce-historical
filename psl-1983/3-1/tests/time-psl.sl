% TIME-PSL.SL  Driver of PSL "spectral" tests
% After loading psl-timer.b, LAPIN  this file

(wrs (open "time-psl.out" 'output))
(prin2  "PSL Spectral Tests,  ") (prin2 (versionname)) 
	(prin2 ",  ") (prin2T (date))
(prin2t 
"---------------------------------------------------------------")
(TestSetup)
(progn	(reclaim)
	(prin2 "EmptyTest 10000		")
	(print (TimeEval '(EmptyTest 10000))) 0)
(progn (prin2 "SlowEmptyTest 10000	")
	(print (TimeEval '(SlowEmptyTest 10000))) 0)
(progn (prin2 "Cdr1Test 100		")
	(print (TimeEval '(Cdr1Test 100))) 0)
(progn (prin2 "Cdr2Test 100		")
	(print (TimeEval '(Cdr2Test 100))) 0)
(progn (prin2 "CddrTest 100		")
	(print (TimeEval '(CddrTest 100))) 0)
(progn (prin2 "ListOnlyCdrTest1	")
	(print (TimeEval '(ListOnlyCdrTest1))) 0)
(progn (prin2 "ListOnlyCddrTest1	")
	(print (TimeEval '(ListOnlyCddrTest1))) 0)
(progn (prin2 "ListOnlyCdrTest2	")
	(print (TimeEval '(ListOnlyCdrTest2))) 0)
(progn (prin2 "ListOnlyCddrTest2	")
	(print (TimeEval '(ListOnlyCddrTest2))) 0)
(progn (prin2 "ReverseTest 10		")
	(print (TimeEval '(ReverseTest 10))) 0)
(progn (reclaim)
	(prin2 "MyReverse1Test 10	")
	(print (TimeEval '(MyReverse1Test 10))) 0)
(progn (reclaim)
	(prin2 "MyReverse2Test 10	")
	(print (TimeEval '(MyReverse2Test 10))) 0)
(progn (reclaim)
	(prin2 "LengthTest 100		")
	(print (TimeEval '(LengthTest 100))) 0)
(progn (prin2 "ArithmeticTest 10000	")
	(print (TimeEval '(ArithmeticTest 10000))) 0)
(progn (prin2 "EvalTest 10000		")
	(print (TimeEval '(EvalTest 10000))) 0)
(progn (prin2 "tak 18 12 6		")
	(print (TimeEval '(topleveltak 18 12 6))) 0)
(progn (prin2 "gtak 18 12 6		")
	(print (TimeEval '(toplevelgtak 18 12 6))) 0)
(progn (prin2 "gtsta g0		")
	(print (TimeEval '(gtsta 'g0))) 0)
(progn (prin2 "gtsta g1		")
	(print (TimeEval '(gtsta 'g1))) 0)
(close (wrs NIL))
