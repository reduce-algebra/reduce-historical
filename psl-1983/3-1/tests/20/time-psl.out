PSL Spectral Tests,  DEC-20 test system,  No-Date-Yet
---------------------------------------------------------------
*** Dummy RECLAIM: 9772 Items used, 140228 Items left.
EmptyTest 10000		18
SlowEmptyTest 10000	187
Cdr1Test 100		521
Cdr2Test 100		365
CddrTest 100		268
ListOnlyCdrTest1	1764
ListOnlyCddrTest1	3207
ListOnlyCdrTest2	2708
ListOnlyCddrTest2	4127
ReverseTest 10		458
*** Dummy RECLAIM: 46868 Items used, 103132 Items left.
MyReverse1Test 10	463
*** Dummy RECLAIM: 83532 Items used, 66468 Items left.
MyReverse2Test 10	447
*** Dummy RECLAIM: 120196 Items used, 29804 Items left.
LengthTest 100		554
ArithmeticTest 10000	644
EvalTest 10000		2680
tak 18 12 6		477
gtak 18 12 6		1378
gtsta g0		1133
gtsta g1		1196
